### ISSUE TITLE
broken out of the box?

### ISSUE THREAD

**Issue by denizyuret** [JuliaCI/BenchmarkTools.jl#83]
With Julia 0.6.1 and a fresh install:
```
julia> Pkg.update()

julia> Pkg.add("BenchmarkTools")
INFO: Cloning cache of BenchmarkTools from https://github.com/JuliaCI/BenchmarkTools.jl.git
INFO: Installing BenchmarkTools v0.2.0
INFO: Package database updated

julia> using BenchmarkTools
INFO: Precompiling module BenchmarkTools.

julia> @btime sin(1)
ERROR: UndefVarError: @__MODULE__ not defined

julia> @benchmark sin(1)
ERROR: UndefVarError: @__MODULE__ not defined
```

**Comment by denizyuret** [comment]
Works when `using Compat` used before `using BenchmarkTools`.  Still weird...

**Comment by jrevels** [comment]
Yeah, it's a known bug that you have to load `Compat` into the same namespace. Should be fixed by https://github.com/JuliaCI/BenchmarkTools.jl/pull/77, once I can get around to finishing that PR (or somebody beats me to it).

**Comment by fredrikekre** [comment]
A temporary workaround is to `Pkg.pin("BenchmarkTools", v"0.0.8")`.

**Comment by jrevels** [comment]
closed by #77
